contrib/lkt/language/parser.py

```diff
@@ -2208,6 +2208,41 @@ def expected_type_predicate(expected_type=T.TypeDecl.entity):
     def invalid_expected_type_error_name():
```

Comment:
(for the commit message)

as weel -> as well

from the array elements known types -> from the context-free types for array elements
(or even better: just “for elements”, as we know we’re talking about arrays)

Therefore*
(see https://forum.wordreference.com/threads/therefor-and-therefore.1261337/)